eclass/systemd.eclass

```diff
@@ -45,6 +45,7 @@ _systemd_get_dir() {
 
 	if $(tc-getPKG_CONFIG) --exists systemd; then
 		d=$($(tc-getPKG_CONFIG) --variable="${variable}" systemd) || die
+		d=${d#${EPREFIX}}
```

Comment:
I'd dare say stripping it like this kinda sucks, especially if we readd it later. Maybe we should set a future goal of reversing the logic, i.e. having the function return prefixed path and stripping it in appropriate getters. Maybe not.

Reply:
I had similar thoughts; maybe I will reverse the logic later.